src/encode_lib_genomic.py

```diff
@@ -404,3 +468,83 @@ def remove_read_group(bam, out_dir='.'):
     run_shell_cmd(cmd)
 
     return new_bam
+
+def get_region_size_metrics(peak_file, out_dir='.'):
+    '''
+    From the peak file, return a plot of the region size distribution and
+    the quartile metrics (summary from R)
+    '''
+    import pandas as pd
+    import numpy as np
+    import matplotlib as mpl
+    mpl.use('Agg')
+    from matplotlib import pyplot as plt
+    from collections import OrderedDict
```

Comment:
I separated this `import`s because this is the only function that uses these imported math/plotting packages and there is a bit of dependency problem for these packages... numpy + matplotlib + py2 + py3 + conda/docker things.